python/dask_cudf/dask_cudf/core.py

```diff
@@ -317,27 +317,6 @@ def var(
                 result.divisions = (min(self.columns), max(self.columns))
             return handle_out(out, result)
 
-    def repartition(self, *args, **kwargs):
-        """ Wraps dask.dataframe DataFrame.repartition method.
-        Uses DataFrame.shuffle if `columns=` is specified.
-        """
-        columns = kwargs.pop("columns", None)
-        if columns:
-            warnings.warn(
-                "The column argument will be removed from repartition in "
-                " future versions of dask_cudf. Use DataFrame.shuffle().",
-                DeprecationWarning,
-            )
-            warnings.warn(
-                "Rearranging data by column hash. Divisions will lost. "
-                "Set ignore_index=False to preserve Index values."
-            )
-            ignore_index = kwargs.pop("ignore_index", True)
-            return self.shuffle(
-                on=columns, ignore_index=ignore_index, **kwargs
-            )
-        return super().repartition(*args, **kwargs)
```

Comment:
Why was this removed?

Reply:
The only reason this seems to be existing is to check if `columns` exists. Now that we are removing the deprecated params from this function we'll be left with call to super call function(`return super().repartition(*args, **kwargs)`). Hence removed this function

Reply:
cc @rjzamora

Can you confirm this is a deprecated function that we're safe to remove?

Reply:
> Can you confirm this is a deprecated function that we're safe to remove?

Yes - This can be removed as long as it is no longer being used in tpcxbb (which I doubt it is) cc @beckernick

Reply:
We still `df.repartition(columns=col)` quite a bit. If this functionality is fully covered by the Dask `df.shuffle` API, we should be fine to remove it. I'll file an issue to switch to the shuffle API

Reply:
> We still df.repartition(columns=col) quite a bit. If this functionality is fully covered by the Dask df.shuffle API, we should be fine to remove it. I'll file an issue to switch to the shuffle API on TPCxBB

Okay - Glad I checked with you :)

The functionality is indeed the same as `shuffle` (where the API is consistent with `dask.dataframe`)

Reply:
@galipremsagar lets defer removing this to a follow up, maybe just add a #TODO

Reply:
@kkraus14 Done